app/src/main/java/com/example/habitsmasher/ui/dashboard/HabitViewFragment.java

```diff
@@ -70,49 +80,58 @@ public void onClick(View view) {
             }
         });
 
+        //set up the buttons for the days of the week
+        setupDaysOfTheWeekButtons(view);
+
+        return view;
+    }
+
+    private void openHabitEventsView() {
+        // Create Habit Event List view with needed parameters
+        HabitEventListFragment fragment = HabitEventListFragment.newInstance(_habit, _username);
+
+        // Replace current fragment with habit view
+        FragmentTransaction transaction = this.getFragmentManager().beginTransaction().replace(R.id.nav_host_fragment_activity_main, fragment);
+        transaction.addToBackStack(null);
+
+        // Load new fragment
+        transaction.commit();
```

Comment:
This way opening a new view may be outdated. This may or may not be the same chunk of code that @kaden-dreger was warning @JuliePilz about earlier today.